application/controllers/RegisterController.php

```diff
@@ -333,6 +340,7 @@ public function getTokenId($iSurveyId)
                 $this->aRegisterErrors[] = gT("This email address is already registered but email to that adress could not be delivered.");
             } else {
                 $this->sMailMessage = gT("The address you have entered is already registered. An email has been sent to this address with a link that gives you access to the survey.");
+                $this->sendRegistrationEmail = false;
```

Comment:
But then you don't send registration email and the sentence are clear : 

> "The address you have entered is already registered. **An email has been sent** to this address with a link that gives you access to the survey."

Must add a delay (one hour, by confir.php) .